solr/core/src/test/org/apache/solr/cloud/OverseerCollectionConfigSetProcessorTest.java

```diff
@@ -131,7 +130,7 @@ public class OverseerCollectionConfigSetProcessorTest extends SolrTestCaseJ4 {
   private static HttpClient httpClientMock;
 
   @SuppressWarnings("rawtypes")
-  private static PlacementPluginFactory placementPluginFactoryMock;
+  private final PlacementPluginFactory placementPluginFactory = new SimplePlacementFactory();
```

Comment:
No need to use a mock here (unless I'm missing something), can use an actual implementation.

Reply:
I know this is a matter of opinion, but I recommend using mocks *sparingly*.  I generally avoid altogether.

Reply:
I do agree!